src/ralph/admin/tests/tests_autocomplete.py

```diff
@@ -60,3 +61,17 @@ def test_not_or_empty(self):
             ['name']
         )
         self.assertEqual(len(result), 0)
+
+    def test_autocomplete_endpoint_required_auth(self):
+        url = reverse(
+            'autocomplete-list',
+            kwargs={
+                'app': 'assets',
+                'field': 'service_env',
+                'model': 'BaseObject'
+            }
+        ) + '?q=foobar'
+        resp = self.client.get(url)
+
+        self.assertEqual(302, resp.status_code)
```

Comment:
statuses may be used here instead of hardcoded integer, it's more verbose to use constant IMO

Reply:
After applying @ar4s's suggestion there's no need to check for a code here.